cmd/cmdutil/helpers.go

```diff
@@ -81,6 +86,11 @@ func AddTagFlag(cmd *cobra.Command) {
 	cmd.Flags().StringSliceP("tag", "t", nil, "Adds the specified tag as an additional column to be displayed during the instance selection prompt.")
 }
 
+// AddsAttributeFlag adds the --attribute flag to command.
+func AddAttributeFlag(cmd *cobra.Command) {
```

Comment:
Small addition to add the ability to include additional formatting flags at the selection level, to supplement the `--tags` flag.